src/store/card-slice.jsx

```diff
@@ -87,13 +87,19 @@ export const getInitialCardsAction = () => {
         dispatch(cardSlice.actions.getInitialCards({ initialCards: respData }));
       })
       .catch((error) => {
+        console.log("setErrorMessage", error.message);
         dispatch(
           cardSlice.actions.setErrorMessage({ errorMessage: error.message })
         );
       });
   };
 };
 
+export const loggerMiddleware = (dispatch) => (next) => (action) => {
+  console.log(action.type, action.payload);
+  return next(action);
+};
+
```

Comment:
This is the right way to do this. But the entity is wrong.
Since middlewares are usually some service functions not directly related to state management or whatever, they are placed into a separate folder (`middlewares` or `utils`, for example).